### DISCUSSION TITLE
May I ask the setting of local time stepping?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#759] [Q&A] [answered]
I learned that local time stepping is an efficient method to efficiently calculate non-uniform grids. I saw in SeisSol Documetion that the setting in the parameter file is advised 2, but you can choose 2, 3, 4, 5, 6, 7, 8,9,10,11...etc? When solving the problem that the maximum grid and the minimum grid are very different, is it better to set the ClusteredLTS as large as possible? Where can I find more details about this? Thanks in advance!
![IMG_20221214_013938](https://user-images.githubusercontent.com/114474894/207408483-ddc63763-bbcc-46fa-b44f-c13d671f556b.jpg)

**Comment by krenzland** [comment, marked as answer]
2 should work well for most cases. (Right now, anything except 1/2 is buggy anyway).
